pkg/storage/provider.go

```diff
@@ -131,7 +138,7 @@ func (p *Provider) MakeAvailable(ctx context.Context) (*wdk.TableSettings, error
 // InsertCertificateAuth inserts certificate to the database for authenticated user
 func (p *Provider) InsertCertificateAuth(ctx context.Context, auth wdk.AuthID, certificate *wdk.TableCertificateX) (uint, error) {
 	if auth.UserID == nil || certificate.UserID != *auth.UserID {
-		return 0, fmt.Errorf("access is denied due to an authorization error")
+		return 0, errors.New(ErrMsgAccessDenied)
```

Comment:
instead of just message, create a whole error
and use `fmt.Errorf` because we agreed that we will try to use only `fmt.Errorf` for errors creation even without any format arguments
so please, try to avoid using errors.New